src/redux/categories/Categories.js

```diff
@@ -0,0 +1,24 @@
+// Actions
+const checkStatus = 'CHECK_STATUS';
+
+// intial State
+const initialState = [];
+
+// Reducer
+const categoriesReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case checkStatus:
+      return 'under Construction!';
+    default:
+      return state;
+  }
+};
+
+// Action Creators
+const checkStatusAction = () => ({
+  type: checkStatus,
+});
+
+// Export
+export { checkStatusAction };
+export default categoriesReducer;
```

Comment:
- [ ] Your categories action and reducer should be configured using `createSlice` from redux toolkit. Kindly refactor this code using `createSlice` instead. You can consider looking at [this article](https://www.softkraft.co/how-to-setup-slices-with-redux-toolkit/) for a general overview of the required setup.